scripts/merge-decap-config.ts

```diff
@@ -0,0 +1,45 @@
+import * as fs from "fs";
```

Comment:
This script consolidates our CMS configuration by merging the base config (config-base.yml) with all individual collection files from web/decap-config/collections/.

It outputs the final, merged config.yml to web/public/mgmt/, which is what the CMS will consume.

As with the linting script, we’re also enforcing double quotes on all string values using js-yaml (quotingType: '"', forceQuotes: true) to maintain consistent formatting and reduce config-related noise in diffs.